docs/src/features/index.md

```diff
@@ -2,4 +2,75 @@
 
 🚧 in Progress
 
-Für jeden Service gibt es eine separate Beschreibung zu dessen Aufgaben und Funktionen.
\ No newline at end of file
+Für jeden Service gibt es eine separate Beschreibung zu dessen Aufgaben und Funktionen.
+
+## Fachliches Datenmodell
+
+Die Relationen zwischen den fachlichen Elementen können sich von Wahl zu Wahl unterscheiden.
+Diese Unterschiede sind Ausfluß der gesetzlichen Grundlage der entsprechenden Wahl, sowie des örtlichen 
+Kommunal- und/oder Landes- Wahlrechts.
+
+![Datenmodell:](pictures/vermutetesFachlichesDatenmodell_20241008.png)
+
+`Wahltermin` - ein Datum an dem eine Wahl oder auch mehrere Wahlen oder Abstimmungen im
+gleichen Wahllokal starten. Diese enden in der Regel auch am gleichen Tag, es gibt aber auch
+Wahltermine, die meherere Tage andauern, wie es bei der Kommunalwahl der Fall ist.
+In der Praxis, auch wenn dies seltener der Fall ist, können am gleichen Tag sogar mehrere
```

Comment:
```suggestion
In der Praxis, auch wenn dies seltener der Fall ist, können am gleichen Tag mehrere
```

Reply:
Mit dem "sogar" möchte ich dem Leser einen Impuls verpassen, aus dem bisherigen Lesefluss, in dem er sich befand mit etwas höhere Wachsamkeit die nicht triviale und ( `sogar` :) ) wichtige Tatsache zur Kenntnis zu nehmen, dass es mehrere Wahltermine am gleichen Tag stattfinden können. Es ist nicht so selbstverständlich das anzunehmen, dass an einem "Wahltermin", denn man erst mal mit einem Tag assoziiert, mehrere Termine sein könnten. Ohne "sogar" übermittelt man das Gleiche, aber anders. Der Leser könnte an der Stelle, aufgrund der Nicht-Selbstverständlichkeit der oben erwähnten allgemeinen Annahme über den Begriff "Termin" i.V.m. "Wahl", an dem was er bislang verstanden hat zweifeln. Mit dem "sogar" sagen wir : "Ne ne wundere dich nicht, es ist so, auch wenn es etwas seltsam vorkommt."